indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -1,86 +1,145 @@
+                            <div class="i-box-content">
+                                <div class="right">
+                                    <a class="email-preview-btn"
+                                       id="email-preview-btn-{{ tpl.id }}"
+                                       data-id="{{ tpl.id }}"
+                                       data-visible>
+                                        {%- trans %}Show preview{% endtrans -%}
+                                    </a>
+                                </div>
+                                <div>
+                                    {%- trans %}This email will be sent when the abstract is:{% endtrans -%}
+                                </div>
+                                <ul>
+                                    {% for rule in tpl.rules if rule %}
+                                        <li class="notification-rule">
+                                            <strong>{{ states[rule.state[0]] }}</strong>
+                                            {% if 'track' in rule %}
+                                                {%- trans %}in{% endtrans -%}
+                                                <strong>
+                                                    {% if not rule.track %}
+                                                        {%- trans %}no track{% endtrans -%}
+                                                    {% elif rule.track[0] == '*' %}
+                                                        {%- trans %}any track{% endtrans -%}
+                                                    {% else %}
+                                                        {{ tracks[rule.track[0]].title }}
+                                                    {% endif %}
+                                                </strong>
+                                            {% endif %}
+                                            {% if 'contribution_type' in rule %}
+                                                {%- trans %}as{% endtrans -%}
+                                                <strong>
+                                                    {% if not rule.contribution_type %}
+                                                        {%- trans %}no type{% endtrans -%}
+                                                    {% elif rule.contribution_type[0] == '*' %}
+                                                        {%- trans %}any type{% endtrans -%}
+                                                    {% else %}
+                                                        {{ contrib_types[rule.contribution_type[0]].name }}
+                                                    {% endif %}
+                                                </strong>
+                                            {% endif %}
+                                            {% if not loop.last %}
+                                                {%- trans %} or{% endtrans -%}
```

Comment:
It's not relevant whitespace since you get a space from the linebreak between endif and if anyway so there's no need for extra spacing